.github/actions/dbp-charts/verify-compose/action.yml

```diff
@@ -3,19 +3,22 @@ description: "Setup environment and run docker compose tests"
 inputs:
   compose_file_path:
     description: "full path to docker compose file to test"
-    required: true
-  docker_username:
-    description: "username for Docker Hub"
-    required: true
-  docker_password:
-    description: "password for Docker Hub"
-    required: true
+    required: false
+    default: "./docker-compose.yml"
   quay_username:
     description: "username for Quay.io"
     required: false
   quay_password:
     description: "password for Quay.io"
     required: false
+  # docker hub credentials are only usefull to avoid rate limiting
+  # and must not be used in fork triggered workflow
```

Comment:
that's true for our specific use case only - let's try to be generic here in alfresco-build-tools